docs/docs/cli/reference.md

```diff
@@ -81,6 +81,31 @@ airy config apply [flags]
 ```
 
 
+***
+
+##  Create
+
+Creates an instance of Airy Core
+
+```
+airy create [flags]
+```
+
+#### Options
+
+```
+  -h, --help              help for create
+      --provider string   One of the supported providers (aws|local)
+```
+
+#### Options inherited from parent commands
+
+```
+      --apihost string      Airy Core HTTP API host (default "http://api.airy")
```

Comment:
I'd also split this into `api-host` to keep it consistent

Reply:
I wouldn't do this here. We're reworking the whole config in #1051 and that's the best place for this